jobs/adept_pr/roles/autotested/files/known_failures.txt

```diff
@@ -0,0 +1,8 @@
+# <NVRA>                                   <subtest name>           <human-friendly description>
+docker-1.12.6-55.gitc4618fb.el7.x86_64     docker_cli/run_volumes   should get filtered on rhelatomic7-4-docker peons
+
+docker-1.13.1-19.git27e468e.fc26.x86_64    docker_cli/run_volumes   should get filtered on fedoratomic26-docker-latest peons
+
+docker-*                                   docker_cli/systemd_in_container should be ignored everywhere
+
+docker-latest-42-*                         docker_cli/version       should run normally everywhere
```

Comment:
docker-latest-42??? Is there a versioning change I'm not aware of?

Reply:
lol, nope, just negative testing.